modules/@apostrophecms/doc/ui/apos/apps/AposDoc.js

```diff
@@ -29,10 +34,14 @@ export default () => {
     if (!modal) {
       throw new Error(`${type} is not a valid piece or page type, or cannot be edited`);
     }
+    copyOfId = copyOfId ?? copyOf?._id;
+    if (copyOf && !copyOfId) {
+      throw new Error('copyOf must be either a string or an object with a _id property');
```

Comment:
How about:

`copyOf` (deprecated) must be an object with an `_id` property, if possible pass `copyOfId` instead

Reply:
Changed.